ocaml/xapi/xapi_xenops.ml

```diff
@@ -2597,7 +2597,11 @@ let maybe_cleanup_vm ~__context ~self =
   end
 
 let clear_vusb_attched ~__context ~self =
```

Comment:
Do we actually need this function still?

Reply:
when  vm start failed. we need to clean both attached field of vusb and pusb. But here may be not the better location. We have a ticket to track where it should be. @robhoes  how do you think?

Reply:
Ok, so you mean when vm start failed. the field will be reset , right? make sense , seems don't need this

Reply:
no need this funciton.  Removed.